dotcom-rendering/src/components/ManyNewsletterSignUp.stories.tsx

```diff
@@ -1,9 +1,6 @@
-import { mockRESTCalls } from '../lib/mockRESTCalls';
 import { GroupedNewslettersList } from './GroupedNewsletterList';
 import { ManyNewsletterSignUp } from './ManyNewsletterSignUp.importable';
 
-mockRESTCalls();
```

Comment:
We don't need to add mockFetch to global.fetch here, because that's done in preview.js